lib/gitlab_client.rb

```diff
@@ -0,0 +1,146 @@
+require 'gitlab'
+
+# Monkeypatch the gitlab gem to add support for the endpoint
+# http://doc.gitlab.com/ce/api/notes.html#list-all-merge-request-notes
+class Gitlab::Client
+  module Notes
+    def merge_request_notes(project, merge_request, options={})
+      get("/projects/#{project}/merge_requests/#{merge_request}/notes", :query => options)
+    end
+  end
+end
+
+class GitlabClient
+  def initialize
+    Gitlab.endpoint = 'https://git.panter.ch/api/v3'
```

Comment:
I would add this to `.env` as well in the interest of keeping the code company-agnostic ;-)